translations/catroweb.en.yml

```diff
@@ -779,4 +779,27 @@ tags:
     experimental:
       title: Experimental
     tutorial:
-      title: Tutorial
\ No newline at end of file
+      title: Tutorial
+
+studio:
+  details:
+    general: 'general'
+    projects: 'projects'
+    comments: 'comments'
+    public: 'public'
+    private: 'private'
```

Comment:
Check capitalization

Reply:
will be capitalized through css for 'projects' and 'comments' and 'public' and 'private' have small initials